### PR TITLE
Missing commas

### PR DESCRIPTION
**Pull request by Jlw2uc**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by thecodingaviator** [comment]
Since there has been no activity on this PR, I'm marking this as stale and closing it.
I'm also closing it since the latest version of Bootstrap is now 4.3.

Look forward to more contributions from you in the future!

**Comment by thecodingaviator** [comment]
Since there has been no activity on this PR, I'm marking this as stale and closing it.

Look forward to more contributions from you in the future!

File: guide/english/algorithms/sorting-algorithms/merge-sort/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Since this is a recursion based algorithm, we have a recurrence relation for it.
 
 ``` T(n) = 2 * T(n / 2) + O(n) ```
 
-Putting it in plain english, we break down the subproblem into two parts at every step and we have some linear amount of work that we have to do for merging the two sorted halves together at each step. 
+Putting it in plain english, we break down the subproblem into two parts at every step, and we have some linear amount of work that we have to do for merging the two sorted halves together at each step.
```
Comment (thecodingaviator) [inline guide/english/algorithms/sorting-algorithms/merge-sort/index.md]:
```suggestion
Putting it in plain English, we break down the sub-problem into two parts at every step, we have some linear amount of work that we have to do for merging the two sorted halves together at each step. 
```

Comment (thecodingaviator) [reply guide/english/algorithms/sorting-algorithms/merge-sort/index.md]: Please make these changes

**Comment by RandellDawson** [comment]
@thecodingaviator This one has gone stale.

**Comment by thecodingaviator** [comment]
Hi @Jlw2uc 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️
